Formula/git-recent.rb

```diff
@@ -0,0 +1,20 @@
+class GitRecent < Formula
+  desc "See your latest local git branches, formatted real fancy"
+  homepage "https://github.com/paulirish/git-recent"
+  url "https://github.com/paulirish/git-recent/archive/v1.0.3.tar.gz"
+  sha256 "2ea954f3c1cc3917ad1a0ff5cd361dff7c3f82410bd464a9f5decb0a539155ff"
+
+  def install
+    bin.install "git-recent"
+  end
+
+  test do
+    system "git", "init"
+    system "git", "recent"
+    # User will be 'BrewTestBot' on CI, needs to be set here to work locally
```

Comment:
@MikeMcQuaid Can you give me a hint with the error here, there *is* a space after the `#`, and I can't see what the non-ascii symbol is?! Any way of running those checks locally? (If they do by default, then they pass for me on Sierra.)